src/data-workspace/query-key-factory.js

```diff
@@ -0,0 +1,13 @@
+export const dataValueSets = {
+    byIds: ({ dataSetId, periodId, orgUnitId, attributeOptionCombo }) => [
+        'dataValueSets',
+        {
+            params: {
+                dataSet: dataSetId,
+                period: periodId,
+                orgUnit: orgUnitId,
+                attributeOptionCombo,
```

Comment:
Tbh, I think the api is confusing, it should be:

```js
params: {
    dataSet: {
        id: dataSetId,
    },
```

That's how many of our endpoints and and how it'd make more sense to me as well. Otherwise I'd expect the word "Id" to be in the key to avoid ambiguity

Reply:
Yeah exactly. So would you leave this as is? We could also go with:

```
export const dataValueSets = {
    byIds: ({ dataSet, period, orgUnit, attributeOptionCombo }) => [
        'dataValueSets',
        {
            params: {
                dataSet,
                period,
                orgUnit,
                attributeOptionCombo,
            },
        },
    ],
}
```